parts/kubernetesmasterresources.t

```diff
@@ -316,7 +316,7 @@
             }
           }
 {{if IsVNETIntegrated}}
-          {{range $seq := loop 2 64}}
+          {{range $seq := loop 2 50}}
```

Comment:
Why does it need to be a function? Can we please make it a part of the internal model rather than defining more small template functions?